torch/csrc/jit/register_prim_ops.cpp

```diff
@@ -699,32 +699,57 @@ Operation listEq(const Node* node) {
   };
 }
 
+template <typename T>
+Operation listNe(const Node* node) {
+  return [=](Stack& stack) {
+    T a;
+    T b;
+    pop(stack, a, b);
+    push(stack, a->elements() == b->elements() ? true : false);
```

Comment:
This doesn't conform to Python semantics, which uses lexicographic ordering on lists.

Reply:
How exactly does this not conform? `Tuples and lists are compared lexicographically using comparison of corresponding elements. This means that to compare equal, each element must compare equal and the two sequences must be of the same type and have the same length.`